src/linter/block.go

```diff
@@ -1366,6 +1366,9 @@ func (b *BlockWalker) handleVariable(v ir.Node) bool {
 				w.untrackVarName(varName)
 			}
 		}
+		if IsDiscardVar(varName) && !isSuperGlobal(varName) {
+			b.r.Report(v, LevelError, "discardVar", "Used discard variable $%s value (rename variable if it's intended)", varName)
```

Comment:
"discard value" is our own term which won't be too obvious for the users.

How about this message:
```
Used var that is supposed to be unused (rename variable if it's intended)
```